### PR TITLE
CB-8302 - Add additional logging around DH - DL attaching

### PR DESCRIPTION
**Pull request by gregito**
CB-8302 - Add additional logging around DH - DL attaching

### CODE REVIEW

File: core/src/main/java/com/sequenceiq/cloudbreak/controller/StackCreatorService.java [hunk 1]
```diff
@@ -383,6 +383,7 @@ private Stack prepareSharedServiceIfNeed(Stack stack) {
         if (stack.getDatalakeResourceId() != null) {
             return sharedServiceConfigProvider.prepareDatalakeConfigs(stack);
         }
+        LOGGER.debug("No datalake resource id has been given for stack! (crn: {})", stack.getResourceCrn());
```
Comment (doktoric) [inline core/src/main/java/com/sequenceiq/cloudbreak/controller/StackCreatorService.java:386]: datalake -> Data Lake

File: core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/StackToStackV4ResponseConverter.java [hunk 2]
```diff
@@ -225,22 +223,32 @@ private Long getStackTimeToLive(Stack stack) {
 
     private void addSharedServiceResponse(Stack stack, StackV4Response stackResponse) {
         SharedServiceV4Response sharedServiceResponse = new SharedServiceV4Response();
+        LOGGER.debug("Checking wheter the datalake resource id is null or not for the cluster: " + stack.getResourceCrn());
```
Comment (doktoric) [inline core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/StackToStackV4ResponseConverter.java:226]: whether

File: core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/StackToStackV4ResponseConverter.java [hunk 3]
```diff
@@ -54,6 +40,18 @@
 import com.sequenceiq.cloudbreak.service.stack.StackService;
 import com.sequenceiq.common.api.telemetry.model.Telemetry;
 import com.sequenceiq.common.api.telemetry.response.TelemetryResponse;
+import org.slf4j.Logger;
```
Comment (bergerdenes) [inline core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/StackToStackV4ResponseConverter.java:43]: please change your IDE's settings to follow the same import order as others. Hint: config/idea/idea_settings.jar

Comment (gregito) [reply core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/StackToStackV4ResponseConverter.java]: done

**Comment by doktoric** [comment]
I think this can go 2.28 line because it is not really a blocker

**Comment by gregito** [comment]
> I think this can go 2.28 line because it is not really a blocker

Have changed it to master

**Pull request merged**